src/ctia/schemas/graphql/helpers.clj

```diff
@@ -344,15 +366,16 @@
     interfaces
     fields
     registry]
-   (or (get @registry object-name)
-       (let [builder (-> (GraphQLObjectType/newObject)
-                         (.description description)
-                         (.name object-name)
+    (get-or-update-type-registry
+      registry
+      object-name
+      #(let [builder (-> (GraphQLObjectType/newObject)
+                         (.description ^String description)
+                         (.name ^String object-name)
```

Comment:
Please temporarily permit the funny indentation, a forthcoming commit will wrap this entire return value in a function and restore some order.